prac_09/unreliable_car.py

```diff
@@ -7,20 +7,23 @@
 
 """
 import random
-
 from prac_09.car import Car
-from random import randint
-
 
 class UnreliableCar(Car):
     def __init__(self, name, fuel, reliability):
+        """Initialise an unreliable car instance.
+            reliability is a float, the chance it will drive
+        """
         super().__init__(fuel, name)
         self.reliability = reliability
 
     def __str__(self):
+        """Returns a string like car but with reliability"""
         return f"{super().__str__()}, Reliability:{self.reliability}"
 
     def drive(self, distance):
+        """Drive the car like the car class, with the addition of checking
+        if it will drive or not"""
         will_it_drive = random.randint(0, 101)
```

Comment:
The randint method is inclusive of both end points, so in this instance it is possible for randint to return 101, instead of 100 for 100% reliability.